### DISCUSSION TITLE
Powering from main

### DISCUSSION THREAD

**Discussion by JR2012thman** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#146] [Q&A]
I am not an electrician but I was wondering if there was any way to power this off the breakers themselves? I have seen other products that do this but I don't know how feasible this is. I saw another post where they set up an external box with a mounted power supply that looks like it came from the breakers but I don't have an outlet anywhere near my breaker panel on the outside of my home. If this is a terrible idea please let me know.

**Comment by CircuitSetup** [comment]
That can be done, yes. You would still need a voltage transformer to bring the voltage down to 9v. The one available on CircuitSetup.us will work, but you would have to open the transformer case, cut off the plug prongs, and run the 2 wires to a dedicated 15A breaker. 

If you've never done this before or aren't comfortable with high voltage electricity, I would highly recommend hiring an electrician to add the circuit in your panel. 

I would recommend mounting the meter outside of your panel per the NEC (you're not supposed to mix high voltage and low voltage/DC in a panel)

**Reply by JR2012thman** [reply]
Thanks, I was planning to have this in a separate box for sure. I will take a look at the transformer on circuitsetup.us, I appreciate the quick follow up.
